.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "libproject/utils"]
 	path = libproject/utils
 	url = https://github.com/dmohs/project-management.git
+[submodule "firecloud-tools"]
+	path = firecloud-tools
```

Comment:
Should this exist in the API directory rather than the top level folder? Do we expect to be using these methods from anywhere other than our API?

Reply:
I'm not sure. I could imagine for example packaging up stuff from here with our notebook libraries (if we wanted to interact with FC directly from a notebook.)

I also sorta like having all the submodules at the top level from an organization perspective.

Reply:
Sounds reasonable. I'm convinced. Just wanted to make sure it was considered.